<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Quote</class>
 <widget class="QWidget" name="Edit_Quote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>271</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>464</width>
    <height>271</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Книга:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="book_name">
         <property name="minimumSize">
          <size>
           <width>371</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Название книги</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>161</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe Script</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Старая цитата:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="OldQuote">
           <property name="minimumSize">
            <size>
             <width>191</width>
             <height>111</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>191</width>
             <height>111</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>201</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>201</width>
             <height>16</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe Script</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Измененная цитата:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="NewQuote">
           <property name="minimumSize">
            <size>
             <width>191</width>
             <height>111</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>191</width>
             <height>111</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="chenge">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>23</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="del">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>23</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="back">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>23</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
